zh_cn.lang

```diff
@@ -0,0 +1,591 @@
+gui.xaero_entity_category_items=物品
+gui.xaero_entity_category_other_entities=其他
+gui.xaero_category_settings=设置
+gui.xaero_category_include_list=包含列表
+gui.xaero_box_category_include_list2=“硬包含”中包括的实体顶部的类别或子类别包含的实体类型、
+gui.xaero_category_exclude_list=排除列表
+gui.xaero_box_category_exclude_list2=已经包含在子类中的实体类型中的需要排除的实体类型的列表。可以同时作为“白名单”和“黑名单”，取决于您选择的排除模式。这不会从实体雷达中移除实体，除非您从根类别“任何”中移除。“显示”设置可以将实体从雷达中移除。
+gui.xaero_category_add_subcategory=添加子类别
+gui.xaero_category_hard_include=硬包含
+gui.xaero_box_category_hard_include2=控制哪些实体自动包含在类别或子类别中，无需手动将其加入列表。
+gui.xaero_category_list_add=加入列表……
+gui.xaero_category_exclude_list_mode=排除模式……
+gui.xaero_category_name=类别名称……
+gui.xaero_category_confirm=确认
+gui.xaero_category_delete=删除类别
+gui.xaero_category_delete_confirm=删除以下类别？
+gui.xaero_category_enable_protection=保护：关
+gui.xaero_category_disable_protection=保护：开
+gui.xaero_category_disable_protection_confirm=禁用以下类别的保护？
+gui.xaero_category_enable_protection_confirm=启用以下类别的保护？
+gui.xaero_box_category_protection=类别保护可以让你避免意外删除、移动、更名一个类别或改变该类别的“硬包含”设置。
+gui.xaero_category_protected_category=已保护
+gui.xaero_category_category_move_up=提高优先级
+gui.xaero_category_category_move_down=降低优先级
+gui.xaero_category_delete_list_element=删除
+gui.xaero_category_settings_confirm=确认
+gui.xaero_category_settings_cancel=取消
+gui.xaero_category_settings_cancel_confirm=放弃更改并直接离开？
+gui.xaero_category_settings_reset=重置为默认
+gui.xaero_category_settings_reset_confirm1=重置为默认的类别配置？
+gui.xaero_category_settings_reset_confirm2=自定义的实体类别会被移除，默认的实体类别会被重置。
+gui.xaero_radar_displayed=显示
+gui.xaero_radar_dots_color=点颜色
+gui.xaero_radar_icons_displayed=图标
+gui.xaero_radar_names_displayed=名称
+gui.xaero_radar_render_order=渲染次序
+gui.xaero_box_radar_render_order=配置雷达类别渲染的次序。数字越低越早渲染。但是，在新版的Minecraft中，点始终渲染在图标上方，但此顺序仍可应用于点或图标的渲染。
+gui.xaero_radar_dots_color_team_colour=队伍颜色
+gui.xaero_box_always_entity_nametags2=总是为相同标签的实体显示名称，无论名称设置为什么。
+gui.xaero_radar_setting_displayed=显示雷达
+gui.xaero_radar_setting_names=显示名称
+gui.xaero_radar_setting_icons=显示图标
+gui.xaero_toggle_entity_radar=实体雷达开关
+narration.xaero_ui_list_left_right_usage=按左键和邮件选择该项内的一个列
+gui.xaero_category_help2=选择一个子类别，并前往其设置以配置点颜色、名称、图标等设置。\n \n 多个子类别包含同一实体时，优先选择列表中更高的类别（创建自定义类别时要记住这点）。通常，新的自定义类别应该放在最上面。
+gui.xaero_update_notification=更新通知
+gui.xaero_category_cut=剪切/移动 §e%1$s
+gui.xaero_category_paste=将 §e%1$s§r 粘贴到 §e%2$s
+gui.xaero_category_paste_cancel=取消剪切/粘贴
+gui.xaero_category_duplicate=重复 §e%1$s
+gui.xaero_category_duplicate_confirm=重复以下类别？
+gui.xaero_reverse_entity_radar=倒置雷达渲染顺序
+gui.xaero_adjust_height_for_carpetlike_blocks=调整矮方块的Y
+gui.xaero_box_adjust_height_for_carpetlike_blocks=对于地毯型方块（例如地毯、单层雪、睡莲）的高度值减1。这可以避免这些方块像完整方块那样在地图上显示明显的阴影。
+gui.xaero_box_category=查看/编辑 §e%1$s
+gui.xaero_box_category_settings=§e%1$s§r 的设置
+gui.xaero_auto_convert_wp_distance_km=将距离转化为KM
+gui.xaero_auto_convert_wp_distance_km_never=从不
+gui.xaero_waypoint_distance_precision=路径点距离精度
+gui.xaero_sort_angle=角度距离
+gui.xaero_toggle_enabled=启用
+gui.xaero_toggle_disabled=禁用
```

Comment:
Make sure it means "enabled" and "disabled", not "enable" and "disable", which is the opposite.